### PR TITLE
Refactor psychrometric unit tests

### PR DESCRIPTION
**Pull request by cgsandford**
This is a tidy-up of the unit tests for `WetBulbTemperature`, as some duplication was introduced in the original PR.

Testing:
 - [x] Ran tests and they passed OK

### CODE REVIEW

File: improver_tests/psychrometric_calculations/test_WetBulbTemperature.py [hunk 1]
```diff
@@ -40,119 +40,113 @@
...
-        expected = [[2707271., 2530250., 2348900.]]
+        expected = [2707271., 2530250., 2348900.]
         result = WetBulbTemperature()._calculate_latent_heat(self.temperature)
         self.assertArrayAlmostEqual(result.data, expected)
 
     def test_calculate_mixing_ratio(self):
         """Test mixing ratio calculation"""
-        expected = [[6.06744631e-08, 1.31079322e-03, 1.77063149e-01]]
+        pressure = np.array([1.E5, 9.9E4, 9.8E4], dtype=np.float32)
+        expected = [6.06744631e-08, 1.31079322e-03, 1.77063149e-01]
```
Comment (MoseleyS) [inline improver_tests/psychrometric_calculations/test_WetBulbTemperature.py:69]: Given the default behaviour of `assertArrayAlmostEqual` of `decimal=6`, I don't think the first value is being checked correctly. There also doesn't seem to be an appropriate setting for this whole array. Perhaps this one needs checking individually.

File: improver_tests/psychrometric_calculations/test_WetBulbTemperature.py [hunk 2]
```diff
@@ -40,119 +40,113 @@
...
         self.assertArrayAlmostEqual(result, expected)
 
     def test_calculate_specific_heat(self):
         """Test specific heat calculation"""
-        expected = np.array([[1089.5, 1174., 1258.5]], dtype=np.float32)
+        expected = np.array([1089.5, 1174., 1258.5], dtype=np.float32)
         result = WetBulbTemperature()._calculate_specific_heat(
-            self.mixing_ratio.data)
+            self.mixing_ratio)
         self.assertArrayAlmostEqual(result, expected)
```
Comment (MoseleyS) [inline improver_tests/psychrometric_calculations/test_WetBulbTemperature.py:79]: This needs `decimal=1`

File: improver_tests/psychrometric_calculations/test_WetBulbTemperature.py [hunk 3]
```diff
@@ -40,119 +40,113 @@
...
     def test_values(self):
         """Basic wet bulb temperature calculation."""
-
-        expected = np.array([[185.0, 259.88306, 333.96066]], dtype=np.float32)
         result = WetBulbTemperature().create_wet_bulb_temperature_cube(
             self.temperature, self.relative_humidity, self.pressure)
-
-        self.assertArrayAlmostEqual(result.data, expected)
-        self.assertEqual(result.units, Unit('K'))
+        self.assertArrayAlmostEqual(result.data, self.expected_wbt_data)
```
Comment (MoseleyS) [inline improver_tests/psychrometric_calculations/test_WetBulbTemperature.py:136]: Needs `decimal=3`

File: improver_tests/psychrometric_calculations/test_WetBulbTemperature.py [hunk 4]
```diff
@@ -40,119 +40,113 @@
...
         self.temperature.convert_units('celsius')
         self.relative_humidity.convert_units('1')
         self.pressure.convert_units('kPa')
-
-        expected = np.array([[185.0, 259.88306, 333.96066]], dtype=np.float32)
         result = WetBulbTemperature().create_wet_bulb_temperature_cube(
             self.temperature, self.relative_humidity, self.pressure)
-
-        self.assertArrayAlmostEqual(result.data, expected)
+        self.assertArrayAlmostEqual(result.data, self.expected_wbt_data)
```
Comment (MoseleyS) [inline improver_tests/psychrometric_calculations/test_WetBulbTemperature.py:145]: Needs `decimal=3`

File: improver_tests/psychrometric_calculations/test_WetBulbTemperature.py [hunk 5]
```diff
@@ -181,73 +175,51 @@ def _make_multi_level(cube, time_promote=False):
...
         """Basic wet bulb temperature calculation as if calling the
         create_wet_bulb_temperature_cube function directly with single
         level data."""
-
-        expected = np.array([[185.0, 259.88306, 333.96066]], dtype=np.float32)
         result = WetBulbTemperature().process(
             self.temperature, self.relative_humidity, self.pressure)
-
-        self.assertArrayAlmostEqual(result.data, expected)
+        self.assertArrayAlmostEqual(result.data, self.expected_wbt_data)
```
Comment (MoseleyS) [inline improver_tests/psychrometric_calculations/test_WetBulbTemperature.py:184]: Needs `decimal=3`

File: improver_tests/psychrometric_calculations/test_WetBulbTemperature.py [hunk 6]
```diff
@@ -181,73 +175,51 @@ def _make_multi_level(cube, time_promote=False):
...
         pressure = self._make_multi_level(self.pressure)
-        expected = np.array([[185.0, 259.88306, 333.96066]], dtype=np.float32)
-
         result = WetBulbTemperature().process(
             temperature, relative_humidity, pressure)
-
-        self.assertArrayAlmostEqual(result.data[0], expected)
-        self.assertArrayAlmostEqual(result.data[1], expected)
+        self.assertArrayAlmostEqual(result.data[0], self.expected_wbt_data)
+        self.assertArrayAlmostEqual(result.data[1], self.expected_wbt_data)
```
Comment (MoseleyS) [inline improver_tests/psychrometric_calculations/test_WetBulbTemperature.py:196]: Needs `decimal=3`

**Review by MoseleyS — CHANGES_REQUESTED** [review]
This looks ok. I've just noticed that the AlmostEqual calls need the `decimal` keyword to make them appropriately strict.

**Review by MoseleyS — APPROVED** [review]
In discussion, we decided to leave the `decimal=?` issue to a separate piece of work.

**Review by neilCrosswaite — APPROVED** [review]

**Comment by cgsandford** [comment]
Rebased parent branch which was a mistake.  Have copied changed test file across to original PR and am closing this now.

**Pull request closed without merging**